.github/workflows/conda-build.yml

```diff
@@ -22,7 +25,7 @@ jobs:
       with:
         auto-activate-base: true
         activate-environment: ""
-        channels: local,conda-forge,defaults
+        channels: local,conda-forge
```

Comment:
looks like we need add `miniconda-version: "latest"` to this action to make it work on macos